lib/sobelow/print.ex

```diff
@@ -2,6 +2,8 @@ defmodule Sobelow.Print do
   @moduledoc false
   alias Sobelow.{Finding, Parse}
 
+  defp puts(data), do: data |> IO.ANSI.format() |> IO.puts()
```

Comment:
Could you add a flag to toggle ANSI output that defaults to `true`, and works something like...

```
defp puts(data), do: data |> IO.ANSI.format(ansi?) |> IO.puts()
```

Reply:
One can use `NO_COLOR=1 mix sobelow`, Elixir support that environment variable OOtB since 1.10. For earlier versions it can be done by `ERL_FLAGS="-elixir ansi_enabled false" mix sobelow`.

Reply:
The problem is that, as-is, output isn't colored by default when Sobelow is installed as an escript.